website/pages/docs/builders/amazon/chroot.mdx

```diff
@@ -127,24 +127,35 @@ Here is a basic example. It is completely valid except for the access keys:
 <Tab heading="HCL2">
 
 ```hcl
+// To make Packer read these variables from the environment into the var object,
+// set the environment variables to have the same name as the declared
+// variables, with the prefix PKR_VAR_.
+
+// You could also hardcode them into the file, but we recommend against that.
+
+// export PKR_VAR_aws_access_key=$YOURKEY
 variable "aws_access_key" {
   type = string
+  // default = "hardcoded_key"
 }
 
+// export PKR_VAR_aws_secret_key=$YOURSECRETKEY
 variable "aws_secret_key" {
   type = string
+  // default = "hardcoded_secret_key"
 }
 
 source "amazon-chroot" "basic-example" {
   access_key = var.aws_access_key
   secret_key =  var.aws_secret_key
+  ami_name = "example-chroot"
   source_ami = "ami-e81d5881"
 }
 
 build {
-  source "sources.amazon-chroot.basic-example" {
-    ami_name = "packer-amazon-chroot {{timestamp}}"
-  }
+  sources = [
+    "source.amazon-chroot.basic-example"
+  ]
```

Comment:
Yup that makes it simpler ! 🙂

Reply:
The other example wasn't working, so I modified it.